docs/content/preview/yugabyte-platform/install-yugabyte-platform/install-software/installer.md

```diff
@@ -258,7 +343,11 @@ Services:
 
```

Comment:
just minor that can be addressed later- the output of the status command is not rendered clearly : https://deploy-preview-20043--infallible-bardeen-164bc9.netlify.app/preview/yugabyte-platform/install-yugabyte-platform/install-software/installer/#service-management